dockstore-client/src/test/resources/Dockstore.cwl

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env cwl-runner
```

Comment:
You probably won't want to check this in directly as a test case. The full workflow will take too long to run as a part of integration tests. 
Instead, make your own workflow demonstrating the same principle that can run in a fraction of the time.